### ISSUE TITLE
BYOL Performance with 40 epoch pretraining

### ISSUE THREAD

**Issue by ildoonet** [deepmind/deepmind-research#76]
https://github.com/deepmind/deepmind-research/tree/master/byol#running-a-short-pre-training-locally

In the case of byol trained on 40 epochs, what performance can you expect from linear evaluation?

**Comment by altche** [comment]
Hi,
When using the provided presets at 40 epochs, we're getting 62% +/- 0.5% top-1 accuracy for the online evaluation (stop_gradient classifier in the pretrain job), and slightly lower (61.5% +/- 0.5) scores when performing the standard linear evaluation in eval_experiment.

**Comment by ildoonet** [comment]
@altche Thanks! If you put those number in README, it will be helpful for many people.
